src/main/kotlin/no/nav/syfo/db/UtsendtVarselDAO.kt

```diff
@@ -75,3 +77,21 @@ fun DatabaseInterface.fetchUtsendtVarselByFnr(fnr: String): List<PUtsendtVarsel>
     }
 }
 
+fun DatabaseInterface.fetchUtsendteVarslerSisteManed(): List<PUtsendtVarsel> {
+    val maxDateStart = Timestamp.valueOf(LocalDate.now().minusDays(SYKEPENGER_SOKNAD_MAX_LENGTH_DAYS).atStartOfDay())
```

Comment:
Blir det riktig å bruke 31 som konstant for utsendte varsler siste måned? Mtp ulik lengde på måneder. Kunne brukt innebygd LocalDate funksjonalitet evnt? minusMonths eller hva det heter :)

Reply:
31 dager er makslengde for sykepengersøknad